VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+* **Year 9:** 300
+* **Year 10:** 300
+
+### Is there a minimum amount of Atoms that must be staked to be a validator?
+
+There is no minimum. The top 100 validator candidates with the highest total stake (where total stake = self-bonded stake + delegators stake) are the validators. 
+
+### How will delegators choose their validators?
+
+Delegators are free to choose validators according to their own subjective criteria. This said, criteria anticipated to be important include:
+* **Amount of self-bonded Atoms:** Number of Atoms a validator self-bonded to its staking pool. A validator with higher amount of self-bonded Atoms has more skin-in-the-game, making it more liable for its actions.
+* **Amount of delegated Atoms:** Total number of Atoms delegated to a validator. A high stake shows that the community trusts this validator, but it also means that this validator is a bigger target for hackers. Indeed, hackers are incentivized to hack bigger validators as they receive a reward proportionate to the stake of the validator they can prove to have compromised. Validators are expected to become less and less attractive as their amount of delegated Atoms grows.
+* **Track record:** Delegators will likely look at the track record of the validators they plan to delegate to. This includes seniority, past votes on proposals, historical average uptime and how often the node was compromised.
+
+Apart from these criteria that will be displayed in the Cosmos wallet’s UI, there will be a possibility for validators to signal a website address to complete their resume. Validators will need to build reputation one way or another to attract delegators. For example, it would be a good practice for validators to have their setup audited by third parties. Note though that the Tendermint Team will not approve or conduct any audit itself.
+
+### Do validators need to be publicly identified?
+
+No, they don’t. Each delegator will value validators based on their own criteria. Validators will be able (and are recommended) to register a website address when they nominate themselves so that they can advertise their operation as they see fit. Some delegators may prefer a website that clearly displays the team running the validator and their resume, while others might prefer anonymous validators with positive track records. Most likely both identified and anonymous validators will coexist in the validator set.
+
+### What are the responsiblities of a validator?
+
+Validators have two main responsibilities:
+* **Be able to constantly run a correct version of the software:** validators need to make sure that their servers are always online and their private keys are not compromised. 
+* **Actively participate in governance:** validators are required to vote on every proposal.
+
+Additionally, validators are expected to be active members of the community. They should always be up to date with the current state of the ecosystem so that they can easily adapt to any change.
```

Comment:
`up-to-date`. Let's decide between `up to date` and `up-to-date`.

Reply:
`up-to-date`